### ISSUE TITLE
Implement virtual filesystem

### ISSUE THREAD

**Issue by skx** [skx/cpmulator#104]
So I've created some custom extensions, for changing the CtrlC handler, the CCP, the console, etc, etc.   These extensions require the user has the CTRLC.COM binary present, as well as CCP.COM, CONSOLE.COM, etc.

It makes no sense to bundle those binaries in `cpm-dist` because they're 100% specific to this emulator.

But of course saying:

* Download cpm-dist.
   * Or have your own collection of binaries.
* Then add my utilties there too.

What would be ideal would be if those files were _always_ present on A:, 100% of the time, and were embedded in the interpreter.  (Perhaps along with SUBMIT.COM?)

To do that I'd need to:

* Move the binaries from `samples/` into "embed/".
* Embed the binaries into our binary.
* Update our File-Find, FileOPen, etc, to take account of them.
* Document this.

It shouldn't be hard, and it would users to build a single standalone binary with whatever things they wanted available on A:, or any other drive.

Perhaps we just add:

* embed/A - embed/P
* And include embed/A/* -  embed/P/*

I guess some things, like zork, would require that we embed more than just *.COM, we'd need *.DAT too.  That means I probably don't want to put the sources to the helpers in the same place.

TLDR; Allow embedding CP/M binaries/files inside our emulator.  Merge them in with local contents at run-time.
